test/sqllogictest/chbench.slt

```diff
@@ -1401,22 +1397,18 @@ ORDER BY su_name
 | ArrangeBy (#0)
 
 %7 =
-| Join %3 %4 %5 %6 (= #11 #33 #39)
-| | implementation = Differential %4.(#0) %6.(#0) %5.(#4) %3.()
-| | demand = (#0, #11..#13, #35, #36, #43)
-| Filter "^co.*$" ~(padchar(#43)), (datetots(#35) > 2010-05-23 12:00:00)
-| Reduce group=(#0, #11, #12, #13)
-| | agg sum(#36)
+| Join %3 %4 %5 %6 (= #0 ((#1 * #2) % 10000)) (= #1 #23 #29)
```

Comment:
There is a new join condition equivalent to `su_suppkey = mod(s_i_id * s_w_id, 10000)`